testutils/factories/draftReferral.ts

```diff
@@ -7,6 +7,10 @@ class DraftReferralFactory extends Factory<DraftReferral> {
     return this
   }
 
+  createdAt(date: Date) {
```

Comment:
I'm not sure this really needs its own trait - normally the traits are used to represent some sort of "state" e.g. "service category assigned", "ready for submission" etc. You can directly pass the `createdAt` arg into the factories' `build` methods.

Reply:
ah i didn't know this. i thought it went quite nicely next to `justCreated`.